src/Dumper.php

```diff
@@ -108,15 +111,33 @@ public function dumpSchema(string $table, Connection $db): void
         $this->dumpNewLine($tableCreationCommand . ';');
         $this->dumpNewLine();
 
+        $this->showDumpingSchemaProgress($table);
+    }
+
+    public function dumpViewSchema(View $view): void
+    {
+        $this->dumpNewLine("-- BEGIN STRUCTURE `{$view->getName()}`");
+        $this->dumpNewLine("DROP VIEW IF EXISTS `{$view->getName()}`;");
+        $this->dumpNewLine('/*!40101 SET @saved_cs_client     = @@character_set_client */;');
+        $this->dumpNewLine('SET character_set_client = utf8mb4;');
+
+        $this->dumpNewLine("CREATE VIEW `{$view->getName()}` AS {$view->getSql()}" . ';');
```

Comment:
Can probably just have a single string.
```suggestion
        $this->dumpNewLine("CREATE VIEW `{$view->getName()}` AS {$view->getSql()};");
```